refextract/app.py

```diff
@@ -107,12 +107,41 @@ def extract_references_from_file_url(args):
             )
         return jsonify({"extracted_references": extracted_references})
 
+    @app.route("/extract_references_from_list", methods=["POST"])
+    @parser.use_args(
+        {
+            "raw_references": fields.List(fields.String, required=True),
+            "journal_kb_data": fields.Dict(required=True),
+        },
+        locations=("json",),
+    )
+    def extract_references_from_list(args):
+        references = args.pop("raw_references")
+        journal_kb_data = args.pop("journal_kb_data")
+        journal_dict = {"journals": journal_kb_data}
+        extracted_references = []
+        try:
```

Comment:
Also should we fail everything if one fails? Maybe it do it gracefully and return the raw